core/src/MessageFactory.cc

```diff
@@ -79,11 +79,11 @@ MessageFactory::MessagePtr MessageFactory::New(
 
```

Comment:
condition above was one of merge conflict problems
in main branch it's in a different order: ".gz_msgs." is second there